addons/web/static/src/scss/form_view.scss

```diff
@@ -342,7 +342,10 @@
         }
 
         .o_form_label {
-            font-weight: normal;
+            // NOTE: I don't know why this was here.
```

Comment:
revert this and add strong around the label?

Reply:
As you prefer, but don't keep commented code.